apps/web/utilities/trpc.ts

```diff
@@ -0,0 +1,29 @@
+import { httpBatchLink } from '@trpc/client';
+import { createTRPCNext } from '@trpc/next';
```

Comment:
can we build it in a way so it's not dependent on nextjs? This might make it harder to make the desktop client in the future

Reply:
Can you review it again, I removed any nextjs deps from it

Reply:
looks ok now :) not sure if we really need it (our apis are really simple) but I don't see anything that could stop us from experimenting plus stronger types are always nice to have